wiki/faq.md

```diff
@@ -42,54 +44,61 @@ The maximum share of new Gridcoins for this project stays the same
+7. Once fixed, you can re-allow new tasks on each project and continue crunching
 
-If you have just begun your BOINC journey it is quite possible your work
-units haven't been verified by the necessary servers. Don't Panic,
-Gridcoin is a game of patience. Just relax, take a day or two, and see
-if the problem persists. If so, proceed to the following steps.
+Note that if one of the projects you are attached to is not online, you may not be able to update it. You can just remove the project for now and add it back later, it should converge onto the CPID the rest of your projects use. You can always check in the solo crunching wizard to make sure all your CPIDs are the same. If they change, you may have to repeat this process again and make a new beacon if your CPID changes.
 
-To ensure your wallet is fully unlocked, lock your wallet with a
-passphrase (make it LONG, a password manager like KeePass is
-recommended) and then unlock it with "for staking only" UNticked.
 
-Next, go to Help -\> Debug Window, go to the Console tab and type
-`advertisebeacon` without the quotes.
+---
+### How can I increase my magnitude or earn more GRC? How are rewards calculated?
 
-This will (if successful) send a small transaction at the cost of
-1 GRC. This should add your CPID with the next [superblock](superblock "wikilink").
+If you just started solo crunching or switched BOINC projects, it will take a week or two for your RAC to "ramp up". 
 
-To see if your CPID is included, go to the Console and type `beaconreport`.
+The GRC you are awarded is based on your magnitude. Your magnitude is based on RAC ("recent average credit") which is a rolling average over the past month. Gridcoin relies on the RAC values provided by the project to calculate your magnitude.
 
-If your CPID is not included, wait for the next superblock. In case your
-CPID isn't included after a few superblocks, try resending your beacon
-(`advertisebeacon`).
+However, you can't just compare your RAC <i>between</i> projects because different projects use different systems for how they allocate credit. Because of this, your magnitude is your RAC earned on a project relative to other Gridcoin crunchers. Your "total magnitude" is the magnitude you have earned on each project, added together.
 
-Lock your wallet again, but this time unlock it with "for staking only"
-ticked.
+You can see estimated magnitude levels for your hardware here: http://quickmag.ml/cgi-bin/script.cgi. Unfortunately, data for some whitelisted projects like WCG are not available so they won't be included in the estimate.
+
+See <a href="/guides/optimization.htm">our guide for picking optimal projects</a>
+. 
+Try different projects and see whether your hardware can keep up a high
+magnitude. You can review your outcome magnitude in the wallet.
+
+Anyway, if you like a project you should still support it, to keep the BOINC
+spirit up.

```

Comment:
> Anyway, if you like a project you should still support it, to keep the BOINC
spirit up.

IMO this doesn't belong here

Reply:
This was in the original FAQ, I can remove it if you'd like

Reply:
Ah, in that case it's fine